modules/ROOT/pages/configure-service-mesh.adoc

```diff
@@ -35,108 +36,144 @@ After the new adapter is ready, the old pod is terminated and all requests are r
++
 `./service-mesh adapter logs --name=<adapter_name> --namespace=<namespace>`
 
+== Create a non-Mule API
+
+If your MuleSoft or non-MuleSoft application does not exist, you must create it before you bind the service to the adapter. 
+
+To create an API, perform the following steps:
+
+. Go to *Anypoint Platform > API Manager* and select the *Manage API* drop-down list. 
+. Select *Create new API*.
++
+The Creating an asset page is displayed.
+. Specify the *Name*, *Asset types* details for the API.  
++
+The Endpoint Confoguration window id displayed, as shown in the following screenshot.
++
+image::create-non-mule-api-service-mesh.png[85%, 85%]
++
+. Specify the required details on the page and click *Save*.
++
+Your API is now created.
 
 == Bind your service to an API on API Manager
-After you set up your namespace, you must bind the service with an existing API to the adapter that you created.
+
+After you provision the adapter, you must bind your service with an existing API to the adapter that you created. The adapter uses the unique API ID of the API for this binding. The Anypoint Platform uses the API ID to identify an application and maintains its configuration. The adapter uses this API ID to fetch policies from the Anypoint Platform and applies to your services that are running in the cluster.
 
 To bind the service with an API, perform the following steps:
 
-. If not exists, create your API in API Manager.
-For more information, see https://docs.mulesoft.com/api-manager/2.x/create-instance-task
-. If your applications were already running in the namespace, restart your application:
-`$ kubectl delete pod -n marketing -l app=campaigns`
+. Obtain the API ID of the API that you want to bind with the service.
+. If your applications were already running in the namespace, restart your application to inject the sidecar in the POD:
++
+`$ kubectl -n <namespace> patch deploy <deployname> --type=json -p='[{"op": "replace", "path": "/spec/template/metadata/labels/service-mesh.mulesoft.com","value":"enable"}]`
++
+In the above command, replace the namespace and the deploy name with your specific details.
 +
 . Verify that the application Kubernetes service port is named following the requirements requested by Istio: https://istio.io/docs/setup/additional-setup/requirements/ .
 . From the Service Mesh installer download location in the Command window, type the following command to bind the adapter with an application:
 +
 [source,text,linenums]
 ----
-./service-mesh binding create \
---name=<replace_with_servicename>-anypoint-security \
---namespace=marketing \same as adapter creation
---adapter=anypoint-service-mesh-adapter \same as adapter creation
---apiId=<replace_with_apiId> \
---serviceName=<replace_with_servicename>
+./service-mesh binding create 
+--name=<servicename>
+--namespace=<namespace name>
+--adapter=<adapter name>
+--apiId=<application ID> 
```

Comment:
Not application ID, please change it to apiID

Reply:
Done!

Reply:
Done!